### PR TITLE
Custom audio player with scrollable transcript 

### PR DESCRIPTION
**Pull request by tariqqrasheedd**
- Customized audio player with custom style.
- Scrollable transcript 
- Author will put two source links of media file, first (.ogg) for play and second (.mp3) for download.
- To display transcript, author will put source of transcript.
- User can download transcript file (.txt / .srt ) and media file (.mp3).

### CODE REVIEW

File: audio/static/html/audio_edit.html [hunk 1]
```diff
@@ -2,8 +2,20 @@
...
       <div class="wrapper-comp-setting">
-        <label class="label setting-label" for="audio_src">Audio Source Location</label>
-        <input class="input setting-input" name="audio_src" id="audio_src" value="{src}" type="text" />
+        <label class="label setting-label" for="audio_src">Audio Source (.ogg)</label>
+        <input placeholder="http://audio.com/my-audio.ogg" class="input setting-input" name="audio_src" id="audio_src" value="{src}" type="text"/>
+      </div>
+    </li>
+    <li class="field comp-setting-entry is-set">
+      <div class="wrapper-comp-setting">
+        <label class="label setting-label" for="audio_src">Audio Source (.mp3)</label>
```
Comment (ziafazal) [inline audio/static/html/audio_edit.html]: `for` attribute should be `audio_src_downloadable `

**Review by ziafazal — CHANGES_REQUESTED** [review]
@tariqqrasheedd overall look good. However in the JS you have used both jquery and native javascript methods. Since jquery support is available in edx-platform you should use jquery where possible.

File: audio/static/html/audio_edit.html (hunk 1 above)
Comment (tariqqrasheedd) [reply audio/static/html/audio_edit.html]: ok

**Comment by zee-pk** [comment]
Looks good to me, please merge.

**Pull request merged**